### ISSUE TITLE
How to set nested objects in "body" parameters?

### ISSUE THREAD

**Issue by sudograce** [theoomoregbee/sails-hook-swagger-generator#41]
I want to set necessary parameters in controller in routes.js.
I already tried different options, but without result. 
I can set body parameters as describe [here](https://github.com/theo4u/sails-hook-swagger-generator#custom-route-configuration), but there is described option only with types that are not an objects (not nested). Attempting to use the "schema" also did not lead to success.

Could you help me understand how to properly use your library to achieve the results described above.

Example of my ajv schema of validation. I need structurally the same result in swagger as it.
```js
    signup: {
      type: "object",
      properties: {
        company: {
          type: "object",
          properties: {
            name: {type: "string", minLength: 1, maxLength: 50},
            email: {type: "string", format: "email", minLength: 0, maxLength: 50}
          },
          required: ["name", "email"]
        },
        contact: {
          type: "object",
          properties: {
            firstName: {type: "string", minLength: 1, maxLength: 50},
            email: {type: "string", format: "email", minLength: 0, maxLength: 50}
          },
          required: ["firstName", "email"]
        }
      },
      required: ["company", "contact"]
    }
```

I would be grateful for any help!

**Comment by theo4u** [comment]
you can follow a similar pattern in https://github.com/theo4u/sails-hook-swagger-generator#custom-route-configuration then move the required array as a field in each of the properties object fields

**Comment by sudograce** [comment]
I tried to use this pattern. But all attempts to configure routes so that embedded objects are displayed are inconclusive unfortunately. The code doesn't convert to the correct swagger.json file.
Could you give me an example, please?
Because now I can get in my ui only it:
![img](url:https://i.ibb.co/KwkVLvk/2018-12-23-19-37-05.png)

Code example (one of the last attempt):
```js
  Route.generate({
    method: "POST",
    endpoint: "auth/signin",
    controller: controller,
    action: "signin",
    swagger: {
      summary: "User login",
      description: "User login",
      parameters: [{
          in: 'body',
          type: 'object',
          name: 'contact',
          schema: {
            type: 'object',
            properties: {
              name: { type: "string", minLength: 1, maxLength: 50 }
            }
          },
          required: true
      }, {
        in: 'body', 
        type: 'object',
        name: 'company',
        schema: {
          type: 'object',
          properties: {
            name: { type: "string", minLength: 1, maxLength: 50 }
          }
        },
        required: true
      }]
    }
  });
```

**Comment by sudograce** [comment]
Also I can't use models as definition (as described in swagger documentation) and I have not opportunity to change custom definition inside routes.js...

**Comment by theoomoregbee** [comment]
should be fixed in #58 else you can drop a comment here
